help/en/html/tools/logixng/Tutorial/chapter9.shtml

```diff
@@ -0,0 +1,253 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <title>LogixNG Tutorial - Chapter 9</title>
+  <meta name="author" content="Daniel Bergqvist">
+  <meta name="author" content="Dave Sand">
+  <meta name="keywords" content="jmri LogixNG tutorial formula">
+
+  <!--#include virtual="/Style.shtml" -->
+
+</head>
+
+<body>
+  <!--#include virtual="/Header.shtml" -->
+
+  <div id="mBody">
+
+    <div id="mainContent" class="no-sidebar">
+      <h1>LogixNG Tutorial - Chapter 9</h1>
+
+      <h2>Formula</h2>
+
+      <p>LogixNG has native support for complex calculations with the tool "Formula".
+      Formula supports almost all the Java operators and you can use local variables and
+      functions with formula. In many cases, like the action Turnout, you can choose to
+      use formula to get the turnout you want to act on, or to get the new state you want
+      to set.</p>
+
+      <p>Local variables, which are explained in <a href="chapter8.shtml">chapter 8</a>,
+      can be used directly in formula. So if you have a local variable <em>index</em>, you
+      can for example have the formula <em>"IT" + str(index)</em>, which adds the string "IT"
+      and the value of <em>index</em>. This can be useful if you for example want to set
+      all the turnouts IT1, IT2, IT3, ..., IT10 to thrown. You can then use the action
+      <strong>For</strong> to iterate from 1 to 10 and to set each of these turnouts to
+      thrown.</p>
+
+      <h3>The Formula expressions</h3>
+
+      <p>There are three expressions for formula: Analog Formula, Digital Formula and
+      String Formula. They all work the same way, except that Analog Formula returns a
+      floating point number, Digital Formula returns a boolean value (true or false), and
+      String Formula returns a string. The expression formula can have child expressions,
```

Comment:
I think it would be good to write `formula` with the first letter as upper case in this sentense:
`The expression Formula can have child expressions`

Reply:
Done.